Core/src/test/java/put/io/black/java/core/logic/NormalNodeTest.java

```diff
@@ -0,0 +1,18 @@
+package put.io.black.java.core.logic;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class NormalNodeTest {
```

Comment:
Missing tests. Add or delete this test class (you have only `setUp()`).

Reply:
NormalNodeTest not need test, i will delete him